build-test.xml

```diff
@@ -12574,6 +12574,9 @@ auto.deploy.dest.dir=C:/WINDOWS/system32/config/systemprofile/liferay/websphere-
 			</then>
 		</if>
 
+		<echo append="true" file="portal-impl/src/portal-ext.properties">
```

Comment:
moved this to the end of file "portal-impl/src/portal-ext.properties", as Victor recommend me to add this properties to this file under `target name="prepare-portal-ext-properties"`